src/auth-service/controllers/request-access.js

```diff
@@ -153,6 +155,7 @@ const requestAccess = {
         }
       }
     } catch (e) {
+      logger.error(`Internal Server Error ${e.message}`);
```

Comment:
In all places where you see `logger.error()`, those messages are also sent to the #notifs-auth-service Slack channel. @BenjaminSsempala @wabinyai @geed254 @Codebmk